kendzi3d-plugin/src/main/java/kendzi/josm/kendzi3d/jogl/model/BarrierFence.java

```diff
@@ -141,66 +143,27 @@ public void draw(GL2 gl, Camera camera, boolean selected) {
     @Override
     public void draw(GL2 gl, Camera camera) {
 
-        enableTransparentText(gl);
-
         // replace the quad colors with the texture
         // gl.glTexEnvi(GL2.GL_TEXTURE_ENV, GL2.GL_TEXTURE_ENV_MODE,
         // GL2.GL_REPLACE);
         gl.glTexEnvi(GL2ES1.GL_TEXTURE_ENV, GL2ES1.GL_TEXTURE_ENV_MODE, GL2ES1.GL_MODULATE);
 
-        gl.glEnable(GL.GL_CULL_FACE);
-
         gl.glPushMatrix();
         gl.glTranslated(getGlobalX(), 0, -getGlobalY());
 
-        try {
-            modelRender.render(gl, model);
-
-            for (RelationCloneHeight cloner : heightClone) {
-                for (Double height : cloner) {
-
-                    gl.glPushMatrix();
-                    gl.glTranslated(0, height, 0);
+        modelRender.render(gl, model);
 
-                    modelRender.render(gl, model);
-                    gl.glPopMatrix();
+        for (RelationCloneHeight cloner : heightClone) {
+            for (Double height : cloner) {
 
-                }
+                gl.glPushMatrix();
+                gl.glTranslated(0, height, 0);
+                modelRender.render(gl, model);
+                gl.glPopMatrix();
             }
-
-        } finally {
```

Comment:
that finally section had some sense, if rendering throws exception we leave with GL_CULL_FACE set. But maybe now it is stable enough :)

Reply:
I had removed the whole try { } construct, because the methods within did not seem to throw any exceptions (at least they declare none being thrown), or did you want to catch exceptions not extending the exception class (e.g. java.lang.OutOfMemoryError)?

What specific exceptions were you trying to handle with that block?